dmass.py

```diff
@@ -1,47 +1,72 @@
 import discord
-from discord.ext.commands import bot
-from discord import game
-from discord.ext import commands
 import asyncio
 import platform
 import colorsys
+
 import random
 import time
+import datetime
 
-client = commands.Bot(command_prefix = '+', case_insensitive=True)
+from discord.ext.commands import bot
+from discord import game

```

Comment:
Lots and Lots of Unnecessary imports,
I recommend using `import discord` instead of `from discord import ...`

Reply:
yes idk about the maintainer

Reply:
Just fix it yourself.